db-engine/src/main/java/com/splicemachine/db/iapi/sql/dictionary/SchemaDescriptor.java

```diff
@@ -122,6 +123,8 @@
      */
     public	static	final	String	STD_DEFAULT_SCHEMA_NAME = Property.DEFAULT_USER_NAME;
 
+	public static final String  STD_SYSTEM_VIEW_SCHEMA_NAME         = "SYSVW";
```

Comment:
nit: remove the additional leading spaces + spaces before `=`

Reply:
Fixed